components/utils/PatientInfoCard.tsx

```diff
@@ -75,6 +77,19 @@ export default function PatientInfoCard({
                 <Edit />
               </Button>
             </Tooltip>
+            <Tooltip label="Copy Patient" openDelay={1000}>
+              <Button
+                size="xs"
+                data-testid="copy-patient-button"
```

Comment:
```suggestion
```
I think we can get rid of the `data-testid` since we use `getByLabelText()` in the unit tests and in general we prefer to use this over using the test id

Reply:
Huh- didn't know we preferred using label text. Good to know! Thanks!